src/main/java/teammates/ui/pagedata/AdminSessionsPageData.java

```diff
@@ -185,10 +185,9 @@ public String getSessionStatusForShow(FeedbackSessionAttributes fs) {
     }
 
     private void setFilter() {
-        filter = new AdminFilter(TimeHelper.formatDate(rangeStart), getHourOptionsAsHtml(rangeStart),
-                                 getMinuteOptionsAsHtml(rangeStart), TimeHelper.formatDate(rangeEnd),
-                                 getHourOptionsAsHtml(rangeEnd), getMinuteOptionsAsHtml(rangeEnd),
-                                 getTimeZoneOptionsAsHtml());
+        filter = new AdminFilter(TimeHelper.formatDateForSessionsForm(rangeStart), getHourOptionsAsHtml(rangeStart),
+                         getMinuteOptionsAsHtml(rangeStart), TimeHelper.formatDateForSessionsForm(rangeEnd),
+                         getHourOptionsAsHtml(rangeEnd), getMinuteOptionsAsHtml(rangeEnd), getTimeZoneOptionsAsHtml());
```

Comment:
Should be 8 spaces from the start of `filter`. Might look better if the first two parameters are on the next line from the constructor call, aligned with the rest of the parameters.

Reply:
Is it possible to add these indentation rules in checkstyle test?

Reply:
Probably possible. You are free to open an issue :)